adoc/Private-Registry-Installation.adoc

```diff
@@ -586,8 +578,6 @@ spec:
   sentinel:
     replicas: 3
     image: registry.suse.com/harbor/harbor-redis:2.1.1
-    customConfig:
-      - "dir /data"
```

Comment:
So why is this not needed after all?

Reply:
@jsuchome this setting is only valid for the `redis` pods, not for the `sentinel` ones. Harmless, but this does cause some warnings to be logged in the sentinel pods.